dotnet/ComputerVision/AnalyzeImage/Program.cs

```diff
@@ -0,0 +1,153 @@
+﻿namespace Microsoft.Azure.CognitiveServices.Samples.ComputerVision.AnalyzeImage
+{
+    using Newtonsoft.Json.Linq;
```

Comment:
Can you move these using statements to outside the namespace, similar to how it is in the SDK? Then repeat to all the programs.